x-pack/solutions/observability/plugins/observability_ai_assistant_app/scripts/evaluation/kibana_client.ts

```diff
@@ -86,13 +86,14 @@ export interface ChatClient {
 export class KibanaClient {
   axios: AxiosInstance;
   constructor(
-    private readonly log: ToolingLog,
+    public readonly log: ToolingLog,
     private readonly url: string,
     private readonly spaceId?: string
   ) {
     this.axios = axios.create({
       headers: {
         'kbn-xsrf': 'foo',
+        'x-elastic-internal-origin': 'observability-ai-assistant',
```

Comment:
these are already added on 131. I think adding them here is fine, but then we'll probably want to remove those headers below.